documentation/book/con-what-the-topic-operator-does.adoc

```diff
@@ -8,8 +8,6 @@ include::common/attributes.adoc[]
 = What the Topic Operator does
```

Comment:
I'm not native speaker ... but _What the Topic Operator does_ sounds strange to me. Why not just _What is Topic Operator_?

Reply:
Yes, there is a word missing :)

Reply:
I've struggled with this one: to me "What the Topic Operator does" sounds slightly more natural than "What the Topic Operator is". We will anyway say what it _is_ by describing what it _does_. The user is ultimately interested in what it _does_ (and thus helping them decide whether it's useful to them), rather than what it _is_. I'll rename it if you really want though.

Reply:
"does" works for me for now. I'm going to try and work on some section headings once we have all the pieces.